src/rpdk/core/jsonutils/inliner.py

```diff
@@ -43,7 +45,16 @@ def _walk(self, obj, old_path):
                     if old_path in self.ref_graph:
                         LOG.debug("Already visited '%s' (%s)", old_path, value)
                         return
-                    url, resolved = self.resolve(value)
+                    try:
+                        url, resolved = self.resolve(value)
+                    except RefResolutionError:  # noqa: E203
+                        if value.contains(self.META_SCHEMA):
+                            url, resolved = self.resolve(  # noqa: E203
+                                value[len(self.META_SCHEMA) :]  # noqa: E203
+                            )  # noqa: E203
+                        else:  # noqa: E203
+                            raise RefResolutionError  # noqa: E203
```

Comment:
If you write this, you re-raise the original exception, which gives easier debugging
```suggestion
                            raise
```